src/avram/migrator/columns/json_column.cr

```diff
@@ -0,0 +1,18 @@
+require "./base"
+
+module Avram::Migrator::Columns::JSON
+  class AnyColumn < Base
```

Comment:
you clever guy you. It took me a sec to figure out how this one would work.

Reply:
Hehe thanks. Makes me wonder if we should instead use just `JSON` as the column? `add meta : JSON`?

Reply:
We'd end up having to do some mutation stuff with saving and pulling data out most likely. Since things will return `JSON::Any`. It would be nice if it was shorter, but that could lead to some other issues.

Reply:
That makes sense! Let’s keep it as is then. But maybe we can add a JSONColumn class that if used raises an error telling you to use JSON::Any just in case people forget